pkg/server/singleprocess/service_server_test.go

```diff
@@ -0,0 +1,69 @@
+package singleprocess
```

Comment:
These used to be in service_config_test.go, but they only call service_server apis.